patches/chrome-browser-android-intercept_navigation_delegate_impl.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/android/intercept_navigation_delegate_impl.cc b/chrome/browser/android/intercept_navigation_delegate_impl.cc
+index d654b01eb8063d4fc3978d12313b80eb9b8452c8..a199dfc836adfacf65348d6cba0b27046a0d7ce4 100644
+--- a/chrome/browser/android/intercept_navigation_delegate_impl.cc
++++ b/chrome/browser/android/intercept_navigation_delegate_impl.cc
+@@ -24,6 +24,7 @@ class ChromeInterceptNavigationDelegate : public InterceptNavigationDelegate {
+     NavigationParams chrome_navigation_params(navigation_params);
+     chrome_navigation_params.url() =
+         GURL(net::EscapeExternalHandlerValue(navigation_params.url().spec()));
++    IGNORE_IF_PLAY_VIDEO_IN_BROWSER_IS_AVAILABLE(chrome_navigation_params.url())
```

Comment:
this doesn't need a patch, you can do a chromium_src override of JNI_InterceptNavigationDelegateImpl_AssociateWithWebContents and return BraveInterceptNavigationDelegate

Reply:
I tried it at first and it means replacing whole contents of `intercept_navigation_delegate_impl.cc`. So, I add this define instead of whole change.
If you think, this is fine. I'll follow your decision.